app/flatpak-builtins-config.c

```diff
@@ -63,18 +63,66 @@ looks_like_a_language (const char *s)
   return TRUE;
 }
 
+static gboolean
+looks_like_a_locale (const char *s)
+{
+  gchar **locale = g_strsplit (s, "_", 3);
```

Comment:
why the 3 here ? I think we want
```
g_strsplit (s, "_", 0);;
```
and then add a

```
if (g_strv_length (locale) != 2)
  return FALSE;
```
Also, this should be declared with g_autofree

Reply:
Because this should cover too 3 part locales, like `az_Latn_AZ` which would be latinized azerbaijani, but i like the idea of spliting these up

Reply:
I see